apis/sgv2-restapi/src/main/resources/application.yaml

```diff
@@ -43,6 +43,7 @@ quarkus:
         # we need to define uri templating on our own for now
         # note that order is important
         match-patterns: |
+          /v2/cql,
```

Comment:
Remove

* this is a wrong pattern, see that all use `=`
* we don't need this one, as it has not path based templating